packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/InterventionController.java

```diff
@@ -142,7 +142,8 @@ public ResponseEntity<InterventionPolicy> createIntervention(
 		final Schema.Permission permission =
 				projectService.checkPermissionCanWrite(currentUserService.get().getId(), projectId);
 		try {
-			return ResponseEntity.status(HttpStatus.CREATED).body(interventionService.createAsset(item, permission));
+			return ResponseEntity.status(HttpStatus.CREATED)
+					.body(interventionService.createAsset(item.clone(), permission));
```

Comment:
Ya if you pass in an existing intervention since it has the same id as another it won't let you create one.  So it was either remove the id before we send it or use this clone function which removes the id for us.  I think we do something similar with model config

Reply:
To be consistent, Im deleting the id key on the front end before we create an object

Reply:
Thank you!